docs/index.md

```diff
@@ -43,9 +137,13 @@ Grabble is also a highly _general_ way of storing and manipulating graphs: it is
 # More Reading
 
 ## Reading Material
-- RedisGraph and Matrix-based Graph Algorithms
-- Annotated Hypergraphs
-- graphs vs. Simplicial Complexes Vs. Hypergraphs
+papers I would like to use as tone/template guides 
```

Comment:
Looking through these now. As your writing evolves on this we can refer back to them, but it is hard to assess their connection and impact in isolation/abstraction. Better to think of them in the context of words you actually write down yourself, then we will have a concrete example.